index.html

```diff
@@ -1,8420 +1,5417 @@
-provides the fastest display of PNG images. But unless the viewer
-uses exactly the same display hardware as that used by the author
-of the original image, the colours will not be exactly the same
-as those seen by the original author, particularly for darker or
-near-neutral colours. The <a class="chunk" href="#11cHRM">
-cHRM</a> chunk provides information that allows
-closer colour matching than that provided by <a>gamma</a> correction
-alone.</p>
-
-<p>The <a class="chunk" href="#11cHRM">cHRM</a> data
-can be used to transform the <a>image data</a> from RGB to XYZ and
-thence into a perceptually linear colour space such as CIE LAB.
-The colours can be partitioned to generate an optimal palette,
-because the geometric distance between two colours in CIE LAB is
-strongly related to how different those colours appear (unlike,
-for example, RGB or XYZ spaces). The resulting palette of
-colours, once transformed back into RGB colour space, could be
-used for display or written into a <a class="chunk" href="#11PLTE">
-PLTE</a> chunk.</p>
-
-<p>Decoders that are part of image processing applications might
-also transform <a>image data</a> into CIE LAB space for analysis.</p>
-
-<p>In applications where colour fidelity is critical, such as
-product design, scientific visualization, medicine, architecture,
-or advertising, PNG decoders can transform the <a>image data</a> from
-source RGB to the display RGB space of the monitor used to view
-the image. This involves calculating the matrix to go from source
-RGB to XYZ and the matrix to go from XYZ to display RGB, then
-combining them to produce the overall transformation. The PNG
-decoder is responsible for implementing gamut mapping.</p>
-
-<p>Decoders running on platforms that have a Colour Management
-System (CMS) can pass the <a>image data</a>, <a class="chunk" href="#11gAMA">
-gAMA</a>, and <a class="chunk" href="#11cHRM">
-cHRM</a> values to the CMS for display or further
-processing.</p>
-
-<p>PNG decoders that provide colour printing facilities can use
-the facilities in Level 2 PostScript to specify <a>image data</a> in
-calibrated RGB space or in a device-independent colour space such
-as XYZ. This will provide better colour fidelity than a simple
-RGB to CMYK conversion. The PostScript Language Reference manual
-[[?PostScript]] gives examples. Such decoders
-are responsible for implementing gamut mapping between source RGB
-(specified in the <a class="chunk" href="#11cHRM">
-cHRM</a> chunk) and the target printer. The
-PostScript interpreter is then responsible for producing the
-required colours.</p>
-
-<p>PNG decoders can use the <a class="chunk" href="#11cHRM">
-cHRM</a> data to calculate an accurate greyscale
-representation of a colour image. Conversion from RGB to grey is
-simply a case of calculating the Y (luminance) component of XYZ,
-which is a weighted sum of R, G, and B values. The weights depend
-upon the monitor type, i.e. the values in the
```

Comment:
I am not thrilled about this.
The tool might have a bug?
These lines should have the same indentation.

That said, I would rather accept the bug and have the tool in use. It feels like a net positive to me.